cli/src/commands/extensions/mod.rs

```diff
@@ -30,6 +34,10 @@ pub fn command<'a>() -> Command<'a> {
         .subcommand(
             Command::new("add")
                 .about("Install extension")
+                .arg(
+                    arg!(-y --yes "Automatically accept requested permissions")
+                        .action(ArgAction::SetTrue),
+                )
```

Comment:
Is `action` required here? Flags should be by default `SetTrue`, right?

Also we could consider hiding this option, since "normal" people shouldn't be using it. But I think the documentation appropriately shows already that this is not something usually desired.

Reply:
Not sure we want to hide that: that option would most likely be used when setting up a CI environment in an unsupervised way, so I'd say that's most likely a desired use case.